### PR TITLE
Add Verse (ERC20)

### PR DESCRIPTION
**Pull request by dawndarkness**
This is a follow up PR relating to https://github.com/trustwallet/assets/pull/29222. 

Please find answers to your questions below @Cryptocool1 : 

• What is the purpose of this token? How will it benefit Trust Wallet users?
VERSE is [Bitcoin.com](http://bitcoin.com/)’s ecosystem and rewards token which launched in December 2022. Refer to our whitepaper for details here: https://www.getverse.com/ 
Currently, Trust Wallet users who hold the VERSE token will not see the VERSE token icon and token details in their wallet. By including the VERSE token in this repository, Trust wallet users would be able to see the VERSE token icon and token details in their wallet.

• How is the token being distributed? Are you conducting an airdrop?
VERSE is being distributed through VERSE Farms on the Verse DEX, see: 
https://verse.bitcoin.com/farms/
https://verse.bitcoin.com/staking/eth/
There are no future airdrops planned yet. 

• Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
Private sales for the VERSE token were conducted in December 2022 before the launch of the token.

• Is the token listed on a centralized/decentralized exchange?
The VERSE token is listed on Kucoin, Uniswap, Paraswap, 1inch, BitMart, QuickSwap,  and Verse DEX.

• Is a Smart Contract audit completed? Where can we find this information?
The VERSE Token contract was audited on 27 May 2022
[getverse.com/verse-security-audit.pdf](https://www.getverse.com/verse-security-audit.pdf)

Please let us know if you have anymore questions. Thank you!

### CODE REVIEW

**Comment by cryptomanz1** [comment]
please update your branch with the recently master

File: blockchains/ethereum/assets/0x249cA82617eC3DfB2589c4c17ab7EC9765350a18/info.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "id": "0x249cA82617eC3DfB2589c4c17ab7EC9765350a18",
+  "tags": ["defi"],
+  "links": [
+    {
+      "name": "facebook",
+      "url": "https://facebook.com/buy.bitcoin.news"
+    },
+    {
+      "name": "twitter",
+      "url": "https://x.com/bitcoincom"
```
Comment (cryptomanz1) [inline blockchains/ethereum/assets/0x249cA82617eC3DfB2589c4c17ab7EC9765350a18/info.json:19]: please change: `url: https://twitter.com/bitcoincom`

**Comment by dawndarkness** [comment]
> please update your branch with the recently master

I have rebased and changed the twitter url @cryptomanz1 . The bot has created a lot of comments, do I need to worry about those? Please let me know. Thank you

**Review by 9764319764319764 — COMMENTED** [review]
hamed

**Comment by dawndarkness** [comment]
Hi @cryptomanz1 and @Cryptocool1 I have rebased the PR again. Please let me know if there is anything else you'd like us to do. Thank you

**Comment by shreyanshp** [comment]
@Cryptocool1 @cryptomanz1 

incase this PR looks unclean, please use this branch
https://github.com/bitcoin-portal/trustwallet-assets/tree/add-verse-token

however do we did pay up the merge-fee-bot in older PR, let us know if any more queries please 🙇

**Comment by dawndarkness** [comment]
@cryptomanz1 @Cryptocool1 if this PR looks unclean, please use this branch
https://github.com/bitcoin-portal/trustwallet-assets/tree/add-verse-token
We paid the merge-fee-bot in https://github.com/trustwallet/assets/pull/29222, so let us know if there are any more queries please :bow:

**Comment by cryptomanz1** [comment]
#29393

**Pull request closed without merging**